src/components/AppConfig/HealthCheck.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import { HealthCheckResult } from "@grafana/runtime";
+import { Alert, AlertVariant } from "@grafana/ui";
+
+interface HealthCheckDetails {
+  openAI: OpenAIHealthDetails | boolean;
+  vector: VectorHealthDetails | boolean;
+  version: string;
+}
+
+interface OpenAIHealthDetails {
+  configured: boolean;
+  ok: boolean;
+  error?: string;
+  models: Record<string, OpenAIModelHealthDetails>;
```

Comment:
It could be great to have some comments here explaining the difference between ok and configured. WDYT?

Reply:
Good point, done.